tests/dao_tests/test_questionnaire_response_dao.py

```diff
@@ -1355,7 +1355,7 @@ def test_insert_qr_three_times(self):
             enrollmentStatusParticipantV3_0Time=datetime.datetime(2016, 1, 2),
             enrollmentStatusParticipantV3_1Time=datetime.datetime(2016, 1, 2),
             healthDataStreamSharingStatusV3_1=DigitalHealthSharingStatusV31.NEVER_SHARED,
-            aian=0
+            aian=False
```

Comment:
Can we revert this so it matches entries in the rest of the repo.

Reply:
My tests fail when it is set to 0, not sure how others are passing? Did I merge this incorrectly or something from a recent PR as I did not mess with aian on this ticket.

Reply:
checking on this again now

Reply:
Fixed, so sorry about this accident! ty for catching this @dtharpeuno